.github/workflows/tests.yml

```diff
@@ -206,27 +206,16 @@ jobs:
         working-directory: ${{ env.PLUGIN_DIR }}
         run: composer dump-autoload
 
-      # This ensures that applicable files and folders can be written to by cache plugins.
-      # We only do this on Restrict Content tests, as doing so on wpunit tests results in WordPress
-      # bizarrely attempting to use FTP to write files.
-      - name: Set Permissions for Caching Plugins
-        if: ${{ matrix.test-groups == 'acceptance/restrict-content' }}
+      # This ensures that applicable files and folders can be written to by WordPress and cache Plugins.
+      - name: Set File and Folder Permissions
         run: |
-          mkdir ${{ env.ROOT_DIR }}/wp-content/wpo-cache
-          mkdir ${{ env.ROOT_DIR }}/wp-content/wpo-cache/config
-          sudo chmod g+w ${{ env.ROOT_DIR }}/wp-config.php
+          sudo chmod 767 ${{ env.ROOT_DIR }}/wp-config.php
           sudo chown www-data:www-data ${{ env.ROOT_DIR }}/wp-config.php
-          sudo chmod g+w ${{ env.ROOT_DIR }}/wp-content
+
+          sudo chmod 767 ${{ env.ROOT_DIR }}/wp-content
```

Comment:
Does the user need execute on these files? This chmod includes that.

Reply:
Removing the executable permission results in tests not being able to run, and e.g. log files not being writable/modified:
![Screenshot 2024-01-02 at 14 45 07](https://github.com/ConvertKit/convertkit-wordpress/assets/1462305/1ae8d2a6-ce5f-4aeb-8cbb-84c1161d3066)

These permissions are expansive, but it seems to be the best way to permit the tests and WordPress access to writing to e.g. the Media Library uploads folder ([link](https://github.com/WordPress/gutenberg/issues/22515#issuecomment-658468153))